Sources/Kitura/Kitura.swift

```diff
@@ -23,17 +23,17 @@ import Dispatch
 // MARK Kitura
 
 /**
- A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+ A set of class functions to create, start, and stop Kitura based servers.
  ### Usage Example: ###
- In this example, a function called run is created, inside a server `Application.swift` file. This will create a Kitura server on the specified port, using the given router and run this created server. This `Application.run` function would then be called in your `main.swift` file as a non-returning function to initilize your Kitura server.
+ In this example, a `Router` that sends "Hello world" is initilized. The `Kitura` class then adds a server on port 8080 with that router and is run. The route can then be accessed by viewing http://localhost:8080.
```

Comment:
I don't like the wording "The `Kitura` class then adds a server" as that's not true, you added the server in your sample code. I'm going to ask David Jones to review this description as I am fairly sure that he helped write this initially and I feel that it is important to add more clarity here as it's the first piece of sample code in the API doc within the Kitura class.